pol-core/pol/pol.cpp

```diff
@@ -489,9 +494,9 @@ void handle_resync_request( Network::Client* client, PKTBI_22_SYNC* /*msg*/ )
   client->send_pause();  // dave removed force=true 5/10/3, let uoclient.cfg option determine xflow
                          // packets (else this hangs 4.0.0e clients)
 
-  Core::WorldIterator<Core::MobileFilter>::InVisualRange(
-      client->chr,
-      [&]( Mobile::Character* zonechr ) { send_client_char_data( zonechr, client ); } );
+  Core::WorldIterator<Core::MobileFilter>::InRange( client->chr->pos(), client->chr->los_size(),
```

Comment:
I think this should be a MaxVisualRange + in_range? I believe this should eg. show large multis.

Reply:
It's a MobileFilter so I think InMaxVisualRange is not needed, we need to know here all mobiles which are in range of the current char. The visual size of all characters is 1